package.json

```diff
@@ -5,20 +5,31 @@
   "license": "Apache-2.0",
   "description": "MedMorph Backend Service Client",
   "scripts": {
-    "start": "nodemon ./bin/www",
+    "start": "npm-run-all --parallel startFrontend startBackend",
+    "startBackend": "nodemon ./bin/www",
+    "startFrontend": "webpack",
     "debug": "node --inspect-brk ./bin/www",
     "test": "jest",
     "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
     "lint": "eslint './src/**/*.{js,jsx}'",
```

Comment:
Can you make sure that `lint` and `test` work for both the ui and backend? Seems like test should work without any addtl changes, lint looks like it will need the `./ui/` folder added to args. Also we could theoretically add the ui folder under src right?

Reply:
moving ui under src also makes it part of the lint, so that works out